### PR TITLE
properly add the cname at the right spot

### PR DESCRIPTION
**Pull request by evgeni**
@lzap as one couldn't test it -- it didn't work ;-)

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -131,9 +131,6 @@ jobs:
       - name: Delete all files
         run: rm -rf *
 
-      - name: Add CNAME for docs.theforeman.org
-        run: echo docs.theforeman.org > CNAME
```
Comment (ekohl) [inline .github/workflows/deploy.yml:135]: The alternative to this is to create `public/master/CNAME`. Of course you need to move this to below the `mkdir -p` statement.

Comment (evgeni) [reply .github/workflows/deploy.yml:135]: `public/CNAME`, without the master, but I think just having the file is cleaner?

**Comment by evgeni** [comment]
btw, @lzap there is a `.nojekyll` file in `gh-pages` which probably should get the same treatment? (it's not removed as you don't remove dot files, but still weird).

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (ekohl) [reply .github/workflows/deploy.yml:135]: Yes, probably is.

**Comment by evgeni** [comment]
Seems it did?

**Comment by spetrosi** [comment]
Yup all works! Thank you for fixing this @evgeni

**Comment by ekohl** [comment]
It was added here:
https://github.com/theforeman/foreman-documentation/commit/60c56d312fc0ce6cc85108344934e33847ef1763#diff-adc4bfdb0829dae99e3699393e3fbaa4

**Pull request merged**